win32ss/drivers/miniport/xboxvmp/xboxvmp.c

```diff
@@ -114,6 +115,24 @@ XboxVmpInitialize(
         return FALSE;
     }
 
+    DeviceDescription.ScatterGather = FALSE;
+    DeviceDescription.Dma32BitAddresses = TRUE;
+    DeviceDescription.Dma64BitAddresses = FALSE;
+    DeviceDescription.MaximumLength = 64 * 1024; /* FIXME: specify proper value */
```

Comment:
Why not set this equal to `XboxVmpDeviceExtension->FrameBufferLength`?

Reply:
Ok this one seems to work :)